snappy/services.go

```diff
@@ -46,6 +51,14 @@ type interacter interface {
 // wait this time between TERM and KILL
 var killWait = 5 * time.Second
 
+const (
+	// the default target for systemd units that we generate
+	servicesSystemdTarget = "multi-user.target"
+
+	// the default target for systemd units that we generate
+	socketsSystemdTarget = "sockets.target"
+)
+
```

Comment:
as I said I feel moving them out completely of systemd is a bit extreme, these defs show a bit why...